deploy/roles/connect/tasks/main.yml

```
@@ -105,3 +106,47 @@
     mode: 0644
   tags:
     - django_settings
+
+- name: Cloudwatch agent setup
+  block:
```

My motivation for adding a block here is that the Cloudwatch setup is purely for monitoring and isn't critical for a successful deploy, so if something does go wrong it shouldn't block the deploy.

The biggest thing I imagine that could go wrong is if the file download doesn't succeed then that would essentially block the rest of the Cloudwatch commands.

Curious to hear from other folks if this is perhaps overkill, or if we would rather prefer to fail hard if something went wrong with setting up Cloudwatch.

---

I think we want to fail hard. These are only run when setting up the machine initially. We wouldn't want to run a machine without this monitoring, would rather know and try again. It won't affect anything running because code isn't even deployed on the machines by this point